scrapper.py

```diff
@@ -26,38 +38,104 @@ class Crawler:
     Crawler implementation
     """
     def __init__(self, seed_urls, max_articles: int):
-        pass
+        self.seed_urls = seed_urls
+        self.total_max_articles = max_articles
+        self.urls = []
 
     def _extract_url(self, article_bs):
-        pass
+        class_bs = article_bs.find('div', class_='container-fluid').find_all('li')
+        for link_search_bs in class_bs:
+            self.urls.append('http://www.vestnik.vsu.ru' + link_search_bs.find('a')['href'])
+
 
     def find_articles(self):
         """
         Finds articles
         """
-        pass
+        for seed_url in self.seed_urls:
+            response = requests.get(seed_url, headers=headers)
+            article_bs = BeautifulSoup(response.text, 'html.parser')
+            self._extract_url(article_bs)
 
     def get_search_urls(self):
         """
         Returns seed_urls param
         """
-        pass
+        return self.seed_urls
+
+class HTMLParser:
+    def __init__(self, full_url, i):
+        self.article_url = full_url
+        self.article_id = i
+        self.article = Article(full_url, i)
+
+    def parse(self):
+        article_bs = BeautifulSoup(requests.get(self.article_url, headers=headers).text, 'html.parser')
+        self._fill_article_with_text(article_bs)
+        self._fill_article_with_meta_information(article_bs)
+        self.article.save_raw()
+        return self.article
+
+    def _fill_article_with_text(self, article_bs):
+        table_rows = article_bs.find_all('div', class_='container-fluid')
+        for table_row in table_rows:
+            table_data = table_row.find_all('a')
+            for new_table_data in table_data:
+                if "Текст (PDF)" in new_table_data:
+                    url = new_table_data['href']
+                    pdf_raw_file = PDFRawFile(url, self.article_id)
+                    pdf_raw_file.download()
+                    text = pdf_raw_file.get_text()
+                    parts_of_article = text.split('ЛИТЕРАТУРА')
+                    self.article.text = ''.join(parts_of_article[:-1])
+
+
+    def _fill_article_with_meta_information(self, article_bs):
+        self.article.author = article_bs.find('i').get_text()
+        self.article.title = article_bs.find('b').get_text()
 
 
 def prepare_environment(base_path):
     """
     Creates ASSETS_PATH folder if not created and removes existing folder
     """
-    pass
-
+    try:
+        os.rmdir(ASSETS_PATH)
```

Comment:
will work for empty dictionaries only. read the docs: https://docs.python.org/3/library/os.html#os.rmdir